server/channel_mob.go

```diff
@@ -138,31 +138,12 @@ func (server ChannelServer) mobDamagePlayer(conn mnet.Client, reader mpacket.Rea
 
 }
 
-func (server ChannelServer) mobEffect(conn mnet.Client, reader mpacket.Reader) {
+func (server ChannelServer) mobDistanceFromBoss(conn mnet.Client, reader mpacket.Reader) {
 	/*
 		id := reader.ReadInt32()
 		attack := reader.ReadInt32()
```

Comment:
Should name this distance as we roughly know what it is.

Reply:
I believe this packet only applies to bosses. I found it in WvsGlobal and they do not seem to do anything with it either:
https://github.com/diamondo25/WvsGlobal/blob/master/WvsBeta.Game/Packets/MobPacket.cs#L220